documentation/synthetic_tokens/tutorials/liquidation.md

```diff
@@ -0,0 +1,420 @@
+Specifically, if you want to run a liquidation bot, you need to fund the wallet with 1) synthetic tokens to close liquidated positions, 2) collateral currency to pay the DVM final fee, and 3) Ether to pay transaction fees.
+If you want to run a dispute bot, then this wallet should be funded with 1) collateral currency to pay dispute bonds and the DVM final fee and 2) Ether to pay for transactions.
+
+### Bot private key management
+
+All deployment configurations require a wallet mnemonic (or private key) to be injected into the bots enabling them to submit transactions to preform on-chain liquidations and disputes.
+You can either bring your own mnemonic from an existing wallet or generate a fresh one using the `bip39` package installed within the UMA repo.
+If you have a wallet mnemonic already you can skip this section.
+
+To generate a new mnemonic you can run the following from the `/core` directory:
+
+```bash
+node -e "console.log(require('bip39').generateMnemonic())"
+# your mnemonic should print here
+```
+
+You can then load this mnemonic into truffle and view the associated address.
+To do this set the mnemonic as an environment variable by running:
+
+```bash
+# Add the new mnemonic to your environment variables. Be sure to replace with your mnemonic.
+export MNEMONIC="sail chuckle school attitude symptom tenant fragile patch ring immense main rapid"
+
+# Start the truffle console
+$(npm bin)/truffle console --network kovan_mnemonic
+
+# Print the address of your newly created account
+accounts[0]
+0x45Bc98b00adB0dFe16c85c391B1854B706b7d612
+```
+
+You can now fund this wallet with the associated currency for the type of bot you want to run.
+To learn more about creating synthetic tokens to back fund your liquidation bot see [this](./using_the_uma_sponsor_cli_tool.md) tutorial.
+
+### Creating a price feed API key
+
+All bots require a price feed to inform their liquidation decisions.
+The easiest price feed to intergrade into is [CryptoWatch](https://cryptowat.ch/). To create an API Key do the following:
+
+1. Create an account [here](https://cryptowat.ch/account/create).
+2. Generate an API key [here](Generate an API key).
+
+Keep this key handy. You'll need it when configuring the bots.
+
+## Running the liquidator and disputer bots locally
+
+This tutorial start with running the liquidator and disputer bots locally from your machine without docker or any complex execution environment. This is meant to be the simplest way posable to start up a bot.
+
+**a) Configuring environment**
+To start a bot the first step is to configure the bot's settings.
+Liquidation bots require 4 main configurations settings which are configured using environment variables.
+To set this up create a `.env` file in the `/core` directory of the repo it:
+
+```bash
+POLLING_DELAY=30000
+EMP_ADDRESS=0xDe15ae6E8CAA2fDa906b1621cF0F7296Aa79d9f1
+MNEMONIC=sail chuckle school attitude symptom tenant fragile patch ring immense
```

Comment:
I got a "Mnemonic is invalid" error and when starting the docker containers. I fixed by putting double quotation marks around the mnemonic:

`MNEMONIC="sail chuckle school attitude symptom tenant fragile patch ring immense"`

Reply:
Nevermind not an issue. But as discussed IRL this mnemonic is missing the last two seed words from above